src/components/common/Footer.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const FooterStyle = styled.div`
```

Comment:
In the future, see if we can move styles that are repeated in different components so that they're defined once in the Ant Design theme.